resources/js/processes/scripts/components/ScriptPreview.vue

```diff
@@ -0,0 +1,367 @@
+      this.save(resolve, reject);
+    });
+
+    window.addEventListener("resize", this.handleResize);
+    let userID = document.head.querySelector('meta[name="user-id"]');
+    window.Echo.private(
+        `ProcessMaker.Models.User.${userID.content}`
+    ).notification(response => {
+      this.outputResponse(response);
+    });
+    this.loadBoilerplateTemplate();
+  },
+  beforeDestroy: function() {
+    window.removeEventListener("resize", this.handleResize);
+  },
+
+  methods: {
+    outputResponse(response) {
+      if (this.executionKey && this.executionKey !== response.data.watcher) {
+        return;
+      }
+      ProcessMaker.apiClient.get("scripts/execution/" + response.response.key).then((response) => {
+        if (response.data.exception) {
+          this.preview.executing = false;
+          this.preview.failure = true;
+          this.preview.error.exception = response.data.exception;
+          this.preview.error.message = response.data.message;
+        } else {
+          this.preview.executing = false;
+          this.preview.success = true;
+          this.preview.output = response.data;
+        }
+      });
+
+      if (response.status !== 200) {
+        this.preview.executing = false;
+        this.preview.failure = true;
+        this.preview.error.exception = response.status;
+        this.preview.error.message = response.response;
+      }
+    },
+    stopResizing: _.debounce(function() {
+      this.resizing = false;
+    }, 50),
+    handleResize() {
+      this.resizing = true;
+      this.stopResizing();
+    },
+    execute() {
+      this.preview.executing = true;
+      this.preview.success = false;
+      this.preview.failure = false;
+      this.preview.output = undefined;
+      // Attempt to execute a script, using our temp variables
+      ProcessMaker.apiClient.post("scripts/" + this.script.id + "/preview", {
+        code: this.code,
+        data: this.preview.data,
+        config: this.preview.config,
+        timeout: this.script.timeout
+      }).then((response) => {
+        this.executionKey = response.data.key;
+      });
+    },
+    onClose() {
+      window.location.href = "/designer/scripts";
+    },
+    save(resolve, reject) {
+      ProcessMaker.apiClient
+          .put("scripts/" + this.script.id, {
+            code: this.code,
+            title: this.script.title,
+            description: this.script.description,
+            script_executor_id: this.script.script_executor_id,
+            run_as_user_id: this.script.run_as_user_id,
+            timeout: this.script.timeout,
+            description: this.script.description
+          })
+          .then(response => {
+            ProcessMaker.alert(this.$t("The script was saved."), "success");
+            if (typeof resolve === "function") {
+              resolve(response);
+            }
+          }).catch(err => {
+        if (typeof reject === "function") {
+          reject(err);
+        }
```

Comment:
I dont think this is necessary? 
```suggestion
  console.error(err)
```